common/metrics/defs.go

```diff
@@ -84,6 +84,7 @@ const (
 	StatsTypeTagName   = "stats_type"
 	CacheTypeTagName   = "cache_type"
 	FailureTagName     = "failure"
+	TypeTagName        = "type"
```

Comment:
type is too generic. Use TaskTypeTagName = "task_type"

Reply:
I think that it's better to have generic tags, so they can be reused in other metrics.